includes/assets/web-stories-block/index.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
If this is for the web stories block why doesn't this code live in `assets/src/story-embed-block`? 

Maybe the css files could be in `assets/src/story-embed-block/block/styles/`? Core seems to use a editor.css and style.css pattern. See the [cover](https://github.com/WordPress/gutenberg/tree/master/packages/block-library/src/cover) block.

Reply:
If these are block styles, then then styles should live with the block. Whatever the directory the block is in, these styles should be with it.

Reply:
agree, it's just that I found `embed-block` styles in `includes/assets` so thought to add front-end resources there, as `assets/src` mainly contains code for the editor.

Reply:
There are plans to move that css file into the assets/src, just not got around to it.

Reply:
noted, we can start with the new block assets then 😅